### PR TITLE
[DOC-46/DOC-214/DOC-177/DOC-95] EE doc section overhaul, single distribution bundle update, removal of CLI section

### PR DESCRIPTION
**Pull request by ivan-gravitee**
**Issue**

https://graviteedevops.atlassian.net/browse/DOC-46
https://graviteedevops.atlassian.net/browse/DOC-214
https://graviteedevops.atlassian.net/browse/DOC-177
https://graviteedevops.atlassian.net/browse/DOC-95

**Description**

DOC-46 DOC-214 DOC-177 DOC-95 EE doc section overhaul, single distribution bundle update, removal of CLI section

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/doc-46/index.html)

### CODE REVIEW

File: pages/ee/introduction.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+EE plugins and features for AM that require an EE license:
+
+* link:https://docs.gravitee.io/am/current/am_userguide_mfa_factors_http.html[HTTP Flow Identity Provider]
+* link:https://docs.gravitee.io/am/current/am_userguide_enterprise_identity_provider_cas.html[CAS Identity Provider^] (now shipped separately like all other EE plugins)
+* link:https://docs.gravitee.io/am/current/am_userguide_enterprise_identity_provider_kerberos.html[Kerberos Identity Provider^] (now shipped separately like all other EE plugins)
+* link:https://docs.gravitee.io/am/current/am_userguide_enterprise_identity_provider_saml2.html[SAML 2.0 Identity Provider^]
+* link:https://docs.gravitee.io/am/current/am_userguide_mfa_factors_fido2.html[MFA with FIDO2^]
+* link:https://docs.gravitee.io/am/current/am_userguide_mfa_risk_based.html[Risk-based MFA^]
+* link:https://docs.gravitee.io/am/current/am_userguide_alerts_overview.html[Alerts^] (de facto the option to integrate the link:https://docs.gravitee.io/am/current/am_overview_introduction.html[Alert Engine^] EE module in AM)
```
Comment (phiz71) [inline pages/ee/introduction.adoc:33]: Shouldn't we use asciidoc syntax to create these links ?
Two methods can be used (prefer the first one if possible):
```
link:{% link pages/apim/3.x/installation-guide/installation-guide.adoc %} [here]
link:{{ '/apim/3.x/apim_installguide.html' | relative_url }}[here]
```

**Review by phiz71 — APPROVED** [review]
LGTM

File: pages/ee/introduction.adoc (hunk 1 above)
Comment (ivan-gravitee) [reply pages/ee/introduction.adoc:33]: Thanks @phiz71 - I guess here you mean Jekyll’s [{% link %} tag](https://jekyllrb.com/docs/liquid/tags/#link) :) Not AsciiDoc syntax. I am all for us using the first format as it is the best for sustaining the correct permalinks and respecting the base URL. However, I think it's not worth the time/effort to fix these now, being that we're moving away from Jekyll imminently, and we have a lot of inconsistent usage of links throughout the current docs.

Comment (phiz71) [reply pages/ee/introduction.adoc:33]: 👌🏻 I'm ok with that. You can merge :)

**Comment by ivan-gravitee** [comment]
Pending Alex Drag's review today before merging.

**Pull request merged**